includes/functions.js

```diff
@@ -28,8 +28,77 @@ function yearStartDateToAcademicYearString(year) {
     return `FORMAT_DATE("%G", ${year}) || '-' || FORMAT_DATE("%g", DATE_ADD(${year}, INTERVAL 1 YEAR))`;
 }
 
+/*This function sets out the correct sequence of declarations should be evidenced in by declaration type so that later in the code the actual sequence declarations were evidenced in can be compared to the correct sequence.*/

```

Comment:
Suggest putting the comment inside the function output. That way if someone is viewing the compiled query they can still see the comment.